setup/symfony_server.rst

```diff
@@ -232,6 +232,14 @@ new custom domain.
     Browse the http://127.0.0.1:7080 URL to get the full list of local project
     directories, their custom domains, and port numbers.
 
+You can also add a wildcard domain:
+
+.. code-block:: terminal
+
+    $ symfony proxy:domain:attach *.my-domain
```

Comment:
It might be safer to quote the domain to ensure no shell expansion ever happen.

Reply:
Thanks, it's done :)